### PR TITLE
Remove the HTTPS from the AD FS authorization rule

### PR DESCRIPTION
**Pull request by rotraves-microsoft**
The HTTPS in the authorization rule will cause AD FS to fail in the authorization stage. The right sintax is with HTTP:
"http://schemas.microsoft.com/authorization/claims/permit",

### CODE REVIEW

**Comment by PRMerger13** [comment]
@rotraves-microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by batamig** [comment]
Hi @rotraves-microsoft - thank you for your contribution! 

I don't think we can remove the HTTPS from this article, as we need to include only secure links in our docs.
I also checked both the HTTPS and HTTP URL, and neither of them go anywhere - so it looks like this URL is out of date.
If you have the updated URL, please update it, and otherwise I'll go back to the team for the fix.

Looking forward to hearing from you.

**Comment by rotraves-microsoft** [comment]
This is not a secure link. This is part of a claim rule language that when configured on AD FS using HTTPS, will not work. With the document like, we are providing wrong guidance to our customers.

**Comment by rotraves-microsoft** [comment]
@batamig

**Comment by batamig** [comment]
Thanks @rotraves-microsoft. I'm working on getting approval and confirmation from our team about this one before I sign off.
We appreciate your contribution to docs!

**Review by batamig — APPROVED** [review]

**Comment by batamig** [comment]
#sign-off

**Pull request merged**
